### ISSUE TITLE
pretrain model

### ISSUE THREAD

**Issue by W4ngH4o** [Sense-X/UniFormer#66]
I have downloaded the pretrain model for testing UniFormer-S. Why the top_acc is close to 0?

**Comment by Andy1621** [comment]
Can you provide more details about the model and dataset?
If you test the model on Kinetics, please check this [issue](https://github.com/Sense-X/UniFormer/issues/38) to map the label.

**Comment by W4ngH4o** [comment]
Thanks for your reply. I did have a different kinetics_400_categroies.txt (the contents in it are sorted according to A-Z),maybe that's why the result is close to 0. However, the link of your kinetics_400_categroies.txt is 404, please help me access it. Thanks so much.

**Comment by Andy1621** [comment]
You can simply map the label via [this file](https://github.com/Sense-X/UniFormer/blob/main/video_classification/data_list/k400/kinetics_400_categroies.txt) to check the accuracy.
For my kinetics, please check [here](https://pan.baidu.com/s/1z0BFMKZROuXCI_tLpREKjg). Password: l7s3

**Comment by W4ngH4o** [comment]
Thanks. I remap the label and test the model. However, the top1-acc is 64.22.  I wonder if the input method of data affect the result. In fact, I extracted frames from videos firstly, and then stored in SSD.  Then, it is similar to sampling Something dataset. It's just not random sampling in a segment like TSN, but sampling every 8 frames, if it's 8x8.

**Comment by Andy1621** [comment]
If you change the `dataset.py`, please adopt the same dense sampling strategies as used in pre-trained model. The same frame and same stride.

**Comment by W4ngH4o** [comment]
Thanks so much. I use dataset.py by you to decode videos, and the top1_acc is 74. But I want to know why we need to convert fps to the same fps before frame sampling. And why the target video fps is selected as 30. Thanks for your reply.

**Comment by Andy1621** [comment]
The target video fps 30 is the default fps for Kinetics videos. We use fps30 for a fair comparison.

**Comment by Andy1621** [comment]
As there is no more activity, I am closing the issue, don't hesitate to reopen it if necessary.
